FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageAttachment.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ComposeMessageAttachment.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 24.09.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import UIKit
+import Photos
+
+struct ComposeMessageAttachment {
+    let name: String
+    let size: Int
+    let data: Data
+    let type: String
+}
+
+extension ComposeMessageAttachment {
```

Comment:
Maybe I'm misunderstanding this whole class, but is it really necessary to handle it with different methods type by type? There may be 1000 different file types. I know you mentioned something about using a "simpler method for file picking" so I suppose maybe this is the side effect of that.

In the future, will we will have to handle file types individually like this?

As an example, on browser extension, the browser picks up a file and it doesn't care what type of file it is. The name, data, and mime type string is all it cares.

Reply:
The problem is that ImagePickerDelegate returns different dictionaries with data. E.g. a photo taken from camera doesn't have a name, path and asset but photo that taken from library has all those fields.